bin/publish.sh

```diff
@@ -102,7 +102,7 @@ for action in $actions; do
   case "$action" in
   build)
     echo "go build submodules ${MOD}"
-    CGO_ENABLED="${CGO_ENABLED}" GOOS="${TARGET_GOOS}" GOARCH=${TARGET_GOARCH} go build -o manager.exe
+    CGO_ENABLED="${CGO_ENABLED}" GOOS="${TARGET_GOOS}" GOARCH=${TARGET_GOARCH} go build -o manager
```

Comment:
I don't think this needs to be modified

With .ignore, avoid binary uploads.

Reply:
OK，I will remove it.